include/ada/url_aggregator-inl.h

```diff
@@ -104,10 +103,10 @@ inline void url_aggregator::update_unencoded_base_hash(std::string_view input) {
 }
 
 ada_really_inline uint32_t url_aggregator::replace_and_resize(
-    uint32_t start, uint32_t end, std::string_view input) {
-  uint32_t current_length = end - start;
-  uint32_t input_size = uint32_t(input.size());
-  uint32_t new_difference = input_size - current_length;
+    uint32_t const start, uint32_t const end, std::string_view const input) {
+  uint32_t const current_length = end - start;
+  auto const input_size = static_cast<uint32_t>(input.size());
```

Comment:
You went from

```C++
uint32_t input_size = uint32_t(input.size());
```

to...

```
auto const input_size = static_cast<uint32_t>(input.size());
```

I'd go in the other direction and write...

```C++
uint32_t input_size(input.size());
```

Simpler, easier to read.

Reply:
That would be an implicit cast hidden in it.